package com.core_java.exceptionhandling;

public class Throw_keyword {

	public static void main(String[] args) {

		int i =20;
		int j=0;
		
		try {
			j = 18/i;
			if(j==0)
				throw new ArithmeticException();
				
		}
		catch (ArithmeticException e) {
			e.printStackTrace();
			System.out.println("Exception occured" + e);
		}
		catch (Exception e) {
			System.out.println("Something went worng" + e);
		}
		System.out.println(j);
		System.out.println("bye");
	}

}
